src/reference/sectionC_interpretation.inc

```diff
@@ -7,17 +7,17 @@
 Interpretation of ``import`` elements
 -------------------------------------
 
-#. Each :code:`import units` or :code:`import component` element present in a :ref:`CellML infoset<specA_cellml_infoset>` (the importing infoset) SHALL define a new and distinct instance of the CellML infoset (the imported infoset) that is specified by the parent :code:`import` element's :code:`href` attribute.
+#. Each :code:`import units` or :code:`import component` element present in a :ref:`CellML infoset<specA_cellml_infoset>` (the importing infoset) SHALL define a new and distinct instance of the CellML infoset (the imported infoset) that is specified by the parent :code:`import` element's :code:`href` attribute value.
```

Comment:
This `value` isn't needed for example. The attribute specifies it. It does that through it's value, sure, but we all know that.

The other ones where you say "the attribute is X", there you've got a case for being pedantic and saying "no the attribute _value_ is X". But not here!